src/scene/materials/standard-material.js

```diff
@@ -136,52 +136,53 @@ let _params = new Set();
  * - When anisotropy > 0, anisotropy direction aligns with the bi-normal, and specular anisotropy
  * increases as anisotropy value increases to maximum of 1.
  *
- * @property {number} clearCoat Defines intensity of clear coat layer from 0 to 1. Clear coat layer
+ * @property {number} clearCoat Defines intensity of clearcoat layer from 0 to 1. Clearcoat layer
```

Comment:
Are you sure `clearcoat` is one word? See glTF spec: https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_clearcoat/README.md#overview

Reply:
I had done a quick google and found clearcoat in relatively common usage, for example Wikipedia page https://en.m.wikipedia.org/wiki/Automotive_paint. I could be wrong though!